src/main/java/se/sundsvall/seabloader/integration/db/model/enums/Status.java

```diff
@@ -3,5 +3,6 @@
 public enum Status {
 	UNPROCESSED,
 	PROCESSED,
+	EXPORT_FAILED,
 	FAILED
```

Comment:
Should we rename this to IMPORT_FAILED to better describe the status?

Reply:
Isn't this the failure status for the export (invoiceCache)?

Reply:
I think EXPORT_FAILED is better. It could be depend on parsing failed and then import is'nt in question.